Documentation/ceph-common-issues.md

```diff
@@ -953,3 +954,58 @@ If you are using Kubernetes version is v1.15 or older, you will see an error lik
 >unable to recognize "STDIN": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1"
 >```
 You need to create the CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Note that these pre-1.16 `apiextensions.k8s.io/v1beta1` CRDs are deprecated in k8s v1.16 and will no longer be supported from k8s v1.22.
+
+## Unexpected partitions created
+
+### Symptoms
+**Rook versions v1.6.0-v1.6.6 may create unwanted OSDs on these partitions, which will corrupt
```

Comment:
"these partitions" --> which partitions? since it moved up to the top, maybe needs rewording.